doc/cephadm/services/osd.rst

```diff
@@ -144,6 +144,18 @@ There are a few ways to create new OSDs:
 
     ceph orch daemon add osd host1:data_devices=/dev/sda,/dev/sdb,db_devices=/dev/sdc,osds_per_device=2
 
+* Create an OSD from a specific lvm on a specific host:
```

Comment:
I don't typically see `lvm` used this way, I suggest "Create an OSD on a specific LVM logical volume on a specific host" instead.